src/styles/components/filter-bar.less

```diff
@@ -4,6 +4,10 @@
   flex-wrap: wrap-reverse;
   width: 100%;
 
+  &.flex-wrap {
```

Comment:
We already have `flex-box-wrap`: https://github.com/mesosphere/canvas/blob/0.2.1/layout/foundation.less#L450